install.ps1

```diff
@@ -0,0 +1,56 @@
+If (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(`
+    [Security.Principal.WindowsBuiltInRole] 'Administrator'))
+{
+    Write-Warning “You do not have Administrator rights to run this script!`nPlease re-run this script as an Administrator!”
+    Break
+} else {
+
+    Write-Host "Downloading VirtualBox, VirtualBox Extensions, and Vagrant"
+
+
+curl http://download.virtualbox.org/virtualbox/5.0.20/VirtualBox-5.0.20-106931-Win.exe -O C:\vbox.exe
+curl http://download.virtualbox.org/virtualbox/5.0.20/Oracle_VM_VirtualBox_Extension_Pack-5.0.20-106931.vbox-extpack -O C:\Oracle_VM_VirtualBox_Extension_Pack-5.0.20-106931.vbox-extpack
+curl https://releases.hashicorp.com/vagrant/1.8.1/vagrant_1.8.1.msi -O C:\vagrant.msi
+
+Write-Host "Installing VirtualBox, return to this window once it installs..."
```

Comment:
You can make this even better by skipping if Virtualbox latest (5.0.20) is already installed. And uninstalling older version.